### PR TITLE
Upgrade yarn

### PR DESCRIPTION
**Pull request by jessopb**
https://github.com/lbryio/lbry-desktop/pull/7530 re pr to fix yarn lock

### CODE REVIEW

**Comment by tiziodcaio** [comment]
Thanks, sorry for the slowliness

**Comment by jessopb** [comment]
no worries! thanks for doing the hard work

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -11,27 +11,30 @@ jobs:
     name: lint
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: Borales/actions-yarn@v2.3.0
+      - uses: actions/checkout@v3
+      - run: corepack enable
+      - run: yarn set version berry
```
Comment (tiziodcaio) [inline .github/workflows/deploy.yml:16]: This is not necessary if using corepack... Did the workflows failed?

Comment (jessopb) [reply .github/workflows/deploy.yml:16]: could be moving up to node 16 fixed it, or checking in the .yarn/releases

**Pull request merged**